backends/openvino/quantizer/quantizer.py

```diff
@@ -119,15 +123,18 @@ def __init__(
             compression_mode = mode.value.replace(
                 "wo", ""
             )  # Mode value has to match NNCF CompressWeightsMode
-            weight_compression_configuration = get_weight_compression_configuration(
+            self._wc_config = get_weight_compression_configuration(
                 nncf.CompressWeightsMode(compression_mode),
                 **kwargs,
             )
             subset_size = 1  # Doesn't really matter in this case since it is data-free. Should just be +ve
             self._algo = nncf.quantization.algorithms.weight_compression.algorithm.WeightCompression(
-                subset_size=subset_size, **weight_compression_configuration
+                subset_size=subset_size, **self._wc_config
             )
 
+    def get_wc_algo_configuration(self):
+        return self._wc_config
```

Comment:
General comment: try to avoid returning mutable objects as they are, it is always better to return a copy of a dict/list because it saves code from unexpected data modifications